docs/02.user-guide/user-guide.en.md

```diff
@@ -0,0 +1,1120 @@
+
+If you don't know the ID of the Plan you are interested in, you can also search for it.
+
+If you know the image name, you can use the following command:
+
+```
+knit plan find --image HOST:PORT/IMAGE:TAG
+```
+
+If you know the Tags used in the inputs or outputs, you can use the following:
+
+```
+knit plan find --in-tag KEY:VALUE
+```
+
+```
+knit plan find --out-tag KEY:VALUE
+```
+
+If you want to limit the search to active Plans", use the following command:
+
+```
+knit plan find --active yes
+```
+
+These search conditions can be specified simultaneously, and only the Plans that meet all the conditions will be matched in the search.
+
+Furthermore, `--in-tag` and `--out-tag` can be specified repeatedly, and in that case, the search will only match if there are inputs or outputs that have all the specified "Tags".
+
+When `knit plan find` is executed without any search conditions, it will match all the Plans in the search.
+
+#### Command Line Flags: `knit plan find`
+
+- `--active MODE`: Limits the search to Plans that are active (or inactive).
+    - The following values can be specified for MODE:
+        - `both` (default): Does not consider the active status as a condition.
+        - `yes`, `true`: Searches only for active Plans.
+        - `no`, `false`: Searches only for inactive Plans.
+- `--image IMAGE[:TAG]`: Searches based on the image of the Plan.
+    - `TAG` can be omitted. In that case, the search ignores the tags of the image.
+    - The `IMAGE` part also matches the hostname or port of the registered image in the Plan definition. It does not perform partial matching.
+- `-i KEY:VALUE`, `--in-tag KEY:VALUE`: Limits the search to those with inputs that have the specified Tags.
+    - If this flag is specified multiple times, it only searches for those with inputs that have all the specified Tags.
+- `-o KEY:VALUE`, `--out-tag KEY:VALUE`: Limits the search to those with outputs that have the specified Tags.
+    - If this flag is specified multiple times, it only searches for those with outputs that have all the specified Tags".
+
+### Activating and deactivating a Plan
+
+If you no longer need a Plan, you can deactivate it:
+
+```
+knit plan active no PLAN_ID
+```
+
+By executing this command, the Plan identified by the specified `PLAN_ID` will be deactivated.
+Any Run based on this Plan that has not started yet will be prevented from executing.
+
+You can also reactivate a Plan that was deactivated for some reason.
+
+```
+knit plan active yes PLAN_ID
+```
+
+This command will re-execute any Run based on the Plan with the specified `PLAN_ID` that was previously prevented from executing.
+
+### Updating the compute resources required by a Plan
+
+Let's say you have executed a computational task and found that there is an excess or shortage of compute resources specified in the "Plan" definition. This can be a problem, especially if there is a shortage of resources, as the computation cannot be executed.
```

Comment:
```suggestion
Let's say you have executed a computational task and found that there is an excess or shortage of compute resources specified in the Plan definition. This can be a problem, especially if there is a shortage of resources, as the computation cannot be executed.
```